protocols/raft/src/main/java/io/atomix/protocols/raft/storage/snapshot/impl/DefaultSnapshotStore.java

```diff
@@ -205,20 +176,64 @@ public void delete() {
     }
   }
 
-  /** Creates a memory snapshot. */
-  private Snapshot createMemorySnapshot(final DefaultSnapshotDescriptor descriptor) {
```

Comment:
What does this even mean memory snapshot :sweat_smile:

Reply:
:man_shrugging: